docs/html/development/getting-started.rst

```diff
@@ -118,8 +118,27 @@ To build it locally, run:
 
 The built documentation can be found in the ``docs/build`` folder.
 
+
+REFERENCES(Helping Guide)
+-------------------------
```

Comment:
This seems to intended to act as the heading for the links below -- but those links aren't rendered as text directly, since they're ["External Links"](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links) in reST (reStructuredText, the format that these files are written in).

This heading can be removed.